ocaml/ocaml-pcre2/Portfile

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           ocaml 1.1
+PortGroup           github 1.0
+
+name                ocaml-pcre2
+version             7.5.2
+github.setup        camlp5 pcre2-ocaml ${version}
+github.tarball_from releases
+distname            pcre2-${version}
+extract.suffix      .tbz
+revision            0
+categories          ocaml devel
+maintainers         {pguyot @pguyot} openmaintainer
+license             LGPL
+platforms           darwin
```

Comment:
`platforms darwin` is the default value, we only need to specify platforms if those are different from all Darwin.

Reply:
Man page says it is required, should we update it?

Reply:
Yes.

Reply:
Thanks.
I created a PR for this: [#macports-base/pull/326](https://github.com/macports/macports-base/pull/326)